dali/pipeline/pipeline.cc

```diff
@@ -66,7 +67,14 @@ namespace dali {
                      int default_cuda_stream_priority)
       : built_(false), separated_execution_(false) {
     dali_proto::PipelineDef def;
-    def.ParseFromString(serialized_pipe);
+    //  Reading Protobuf file has a limitation of 64 MB
+    //  Following instructions will increase the
+    //  limits to support the given pipeline
+    google::protobuf::io::CodedInputStream *coded_input =
+        new google::protobuf::io::CodedInputStream(
```

Comment:
aren't you deleting this?

Reply:
@jantonguirao: coded_input is being pased into Pipeline def here. Are you referring to deallocation of coded_input?

Reply:
Done